katello/katello-backup

```diff
@@ -110,6 +183,11 @@ if @dir.nil?
   puts "**** ERROR: Please specify an export directory ****"
   puts optparse
   exit(-1)
+elsif
+  @options[:snapshot] && (@options[:online] || @options[:incremental])
+  puts "**** ERROR: Snapshot backups can only be done offline ****"
```

Comment:
`ERROR: Snapshot backups cannot be done online or incrementally` ?

Also, why can't we do them online?